Resources/Prototypes/_Harmony/Entities/Objects/Weapons/Guns/Projectiles/magic.yml

```
@@ -0,0 +1,21 @@
+﻿- type: entity
+  id: ProjectileBanana
+  name: BURGNANA BLAST
+  description: slip
+  parent: BaseBullet
+  categories: [ HideSpawnMenu ]
```

The ordering is incorrect
```suggestion
  parent: BaseBullet
  id: ProjectileBanana
  categories: [ HideSpawnMenu ]
  name: BURGNANA BLAST
  description: slip
```